privacy-dashboard/privacy-dashboard-api/build.gradle

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'kotlin-android'
+}
+
+apply from: "$rootProject.projectDir/gradle/android-library.gradle"
+
+dependencies {
+    implementation Kotlin.stdlib.jdk7
```

Comment:
I'm not sure this needs to be `implementation`, but happy to change it if it's just to keep consistency or follow any rule.

How I read this is:
- Use `api` if you need dependencies in order to define your API.
- Use `implementation` if you need dependencies for your implementation.

In here, we need those dependencies just because we use them to define our API.

is that correct?